src/pages/Login.vue

```diff
@@ -214,12 +214,17 @@ export default {
         this.requestedOtp = false;
       }
     },
+    requestedOtp()
+    {
+       if (!this.resendOTPTimer) clearInterval(this.interval);

```

Comment:
this should be run if the timer is not = 0 right?

Reply:
For timer we are seeing it below in startCountdown function